### ISSUE TITLE
Renewed support for SSR? 

### ISSUE THREAD

**Issue by cmaycumber** [nandorojo/dripsy#128]
@intergalacticspacehighway recently released: https://github.com/intergalacticspacehighway/rnw-responsive-ssr. Which I thought was pretty interesting. 

Would it be of interest to bring https://github.com/intergalacticspacehighway/react-native-responsive-query as the main driver for creating media queries?

**Comment by nandorojo** [comment]
Yeah it certainly is of interest, I’ve been discussing it with him. It would be a number of breaking changes (for v4) but I think it’s definitely in the cards.

There is now a getter function option returned from the hook, which makes it much more likely we could implement it with Dripsy.

**Comment by intergalacticspacehighway** [comment]
@nandorojo came up with the hook idea and is helping with the API. We're trying to make it customizable enough to be used in RN design systems.
